test/integ/pandas/test_pandas_tools.py

```diff
@@ -417,6 +417,54 @@ def test_write_pandas_create_temp_table_deprecation_warning(
             cnx.execute_string(drop_sql)
 
 
+@pytest.mark.parametrize("use_logical_type", [None, True, False])
+def test_write_pandas_use_logical_type(
```

Comment:
Please add a unit test for verifying the warning you emit

Reply:
The warning is verified in the same test at line 462

Reply:
The warning is tested in the same test at line 462. I could split off testing the warning from the integration test if this is preferred. Other warnings, like the temp_table depreciation warning, are currently also tested during the integration tests, see "test_write_pandas_create_temp_table_deprecation_warning".